<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PAVANKUMAR" failures="2" tests="3" name="IE_Windows10_Test" time="44.264" errors="0" timestamp="12 Jun 2023 10:08:56 GMT">
  <testcase classname="com.lamdatest.tests.LamdaAssesmentTest" name="Scenario_2Test" time="5.258">
    <failure type="java.lang.AssertionError" message="expected [95] but found [51]">
      <![CDATA[java.lang.AssertionError: expected [95] but found [51]
	at com.lamdatest.tests.LamdaAssesmentTest.Scenario_2Test(LamdaAssesmentTest.java:50)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- Scenario_2Test -->
  <testcase classname="com.lamdatest.tests.LamdaAssesmentTest" name="Scenario_3Test" time="4.642">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.lamdatest.tests.LamdaAssesmentTest.Scenario_3Test(LamdaAssesmentTest.java:65)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- Scenario_3Test -->
  <testcase classname="com.lamdatest.tests.LamdaAssesmentTest" name="Scenario_1Test" time="4.645"/>
</testsuite> <!-- IE_Windows10_Test -->
